packages/design-tokens/src/types.ts

```diff
@@ -24,7 +24,7 @@ export type TypographyFont = {
   letterSpacing: KaizenCSSProperties["letterSpacing"]
 }
 
-export type ThemeKey = "zen" | "heart"
+export type ThemeKey = "zen" | "heart" | "custom"
```

Comment:
So `custom` could be either transparent, high-vis, or anything else? Is it intended to be a catch-all for any user-supplied theme?

Reply:
@ActuallyACat yeah exactly. At least for now. We can evolve it later if we need to, perhaps adding an extra string field for the name or description.